packages/react-components/react-button/src/components/Button/Button.tsx

```diff
@@ -12,6 +13,8 @@ export const Button: ForwardRefComponent<ButtonProps> = React.forwardRef((props,
   const state = useButton_unstable(props, ref);
 
   useButtonStyles_unstable(state);
+  const componentStyles = useCustomStyleHooks_unstable();
```

Comment:
nit: I would separate the usage of custom styles in `Button`, in that PR you could also create the documentation, otherwise when we release we might have a customizable button but no documentation on the recommended way to use this... users might get 'creative' 😅

Reply:
I'll have a set of follow-on PRs to this one that will include docs.  I wanted to have Button wired up to show the component consumption side.